### PR TITLE
DEVX-4904 VAPI Configurable Webhook Timeouts

### PR DESCRIPTION
**Pull request by vshisterov**
## Description

Voice API webhook timeouts configuration description added.

### CODE REVIEW

File: _examples/concepts/guides/webhooks-setup/voice.md [hunk 1]
```diff
@@ -47,3 +47,17 @@ Inside an NCCO, the following action types take a webhook URL for use when that
 * [connect.eventUrl](/voice/guides/ncco-reference#connect) - set the URL to the webhook endpoint Nexmo calls asynchronously when a conversation changes state for this connect action
 * [input.eventUrl](/voice/guides/ncco-reference#input) - set the URL to the webhook endpoint Nexmo sends the digits pressed by the callee
 * [stream.streamUrl](/voice/guides/ncco-reference#stream) - set an array of URLs pointing to the webhook endpoints hosting the audio file to stream to the Call or Conversation
+
+### Webhook Timeouts
+
+If the answer, event or fallback URL is unreachable for a certain amount of time, or the response reading time exceeds a certain limit, Vonage will retry the request once. Platform default connection and reading timeouts are as follows:
```
Comment (bencgreenberg) [inline _examples/concepts/guides/webhooks-setup/voice.md:53]: What do you think about shortening this to: "If one of the callback URLs is unreachable.."? @marklewin, thoughts?

Comment (bencgreenberg) [inline _examples/concepts/guides/webhooks-setup/voice.md:53]:
```suggestion
If the answer, event or fallback URL is unreachable for a certain amount of time, or the response time exceeds a certain limit, Vonage will retry the request once. Platform default connection and reading timeouts are as follows:
```

File: _examples/concepts/guides/webhooks-setup/voice.md [hunk 2]
```diff
@@ -47,3 +47,17 @@ Inside an NCCO, the following action types take a webhook URL for use when that
...
+
+Webhook Type | Connect Timeout | Socket Timeout
+-------------|-----------------|----------------
+Answer | 1 second | 5 seconds
+Event | 1 second | 10 seconds
+Fallback | 1 second | 5 seconds
+
+These defaults can be overridden via [Application API](/api/application.v2#updateApplication) call or in the [Dashboard](https://dashboard.nexmo.com/applications) - select the application, then click **Edit** button and scroll to Capabilities / Voice section:
```
Comment (bencgreenberg) [inline _examples/concepts/guides/webhooks-setup/voice.md:61]:
```suggestion
These defaults can be overridden via an [Application API](/api/application.v2#updateApplication) call or in the [Dashboard](https://dashboard.nexmo.com/applications) by selecting the application, then clicking the **Edit** button and scrolling to the Capabilities / Voice section:
```

File: _examples/concepts/guides/webhooks-setup/voice.md (hunk 1 above)
Comment (marklewin) [reply _examples/concepts/guides/webhooks-setup/voice.md]: I think it's OK to be specific here @bencgreenberg ... what do we mean by "response reading time" exactly? Can we explain that better?

Comment (vshisterov) [reply _examples/concepts/guides/webhooks-setup/voice.md]: we have removed the word "reading" to simplify the description (it is indeed just response time).

**Review by bencgreenberg — APPROVED** [review]

**Pull request merged**